ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -62,6 +62,30 @@ open class TabBarItem: NSObject {
                   accessibilityLabelBadgeFormatString: nil)
     }
 
+    /// Initializes `TabBarItem`
+    /// - Parameter title: Used for tabbar item view's label and for its accessibilityLabel.
+    /// - Parameter image: Used for tabbar item view's imageView and for its accessibility largeContentImage unless `largeContentImage` is specified.
+    /// - Parameter selectedImage: Used for imageView when tabbar item view is selected.  If it is nil, it will use `image`.
+    /// - Parameter landscapeImage: Used for imageView when tabbar item view in landscape. If it is nil, it will use `image`. The image will be used in portrait mode if the tab bar item shows a label.
+    /// - Parameter landscapeSelectedImage: Used for imageView when tabbar item view is selected in landscape. If it is nil, it will use `selectedImage`. The image will be used in portrait mode if the tab bar item shows a label.
+    /// - Parameter isUnread: Used for when the tabbar item view has not yet been selected. If never selected, an unread dot will be displayed whent the control is initialized. Default value is false.
+    @objc public convenience init(title: String,
```

Comment:
adding as init of tabbaritem doesn't seem right. unread state is something that we can modify after item is already created. we don't have special init for badgevalue either which matches how eng would setup UITabbaritem too https://developer.apple.com/documentation/uikit/uitabbaritem?language=objc